docs/guide/tech-behind-calypso.md

```diff
@@ -105,28 +57,4 @@ The way we use Git with Calypso is described in the [Git Workflow document](../g
 
 - [lodash](https://lodash.com) – general purpose utility library; includes a ton of useful functions for dealing with arrays, objects, and collections; deprecated because of its lack of modularity and tendency to introduce code smells by implicitly converting different value types and handling nullish (`null` or `undefined`) values.
 
-## Commands executed when you run `yarn start`
```

Comment:
This section doesn't really belong here and there's already a dedicated document for it. I just moved this content to that document instead.